### PR TITLE
Fix #1038: Update description in TopicPractice

### LINKED ISSUE #1038: Update description in TopicPractice
Screenshot_1588568852

Update the description from `Select or unselect…` to `Select skills that you would like to practice.`

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR fixes #1038. It updates string files.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -33,9 +33,9 @@
   <string name="stop_exploration_dialog_description">Your progress will not be saved.</string>
   <string name="stop_exploration_dialog_cancel_button">Cancel</string>
   <string name="stop_exploration_dialog_leave_button">Leave</string>
-  <string name="topic_practice_master_these_skills">Master These Skills</string>
+  <string name="topic_practice_master_these_skills">Master these Skills</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: `Master These Skills`
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/f403add1-079a-4202-9975-efe04c076290/TP-Practice-Unchecked-

**Review by rt4914 — CHANGES_REQUESTED** [review]
One change suggested.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/values/strings.xml]: As per the mock. But I thought it needs to be corrected. Reverted.

Comment (rt4914) [reply app/src/main/res/values/strings.xml]: Can you share that mock link here?

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/values/strings.xml]: > Can you share that mock link here?

No I was thinking there need to be correction or what. Reverted it back.

**Pull request merged**